src/app/tools/service/settings/settings.service.ts

```diff
@@ -0,0 +1,61 @@
+import {Injectable} from '@angular/core';
+import {Location} from '@angular/common';
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+
+import * as shajs from 'sha.js';
+
+import { environment } from '../../../../environments/environment';
+import {Configuration} from './settings.types';
+
+@Injectable()
+export class SettingsService {
```

Comment:
Remove SettingsService as this is internal. Also remove the "settingsConfig" object from the environment files (all of them).